src/morphodict/site/checks.py

```diff
@@ -0,0 +1,28 @@
+from django.conf import settings
+from django.core.checks import Error, register
+
+_MORPHODICT_REQUIRED_SETTING_SENTINEL = object()
+
+
+@register()
+def check_settings(**kwargs):
+    print("running check_settings")
+
+    errors = []
+
+    # borrowed from what django diffsettings command does
+    if not settings.configured:
+        raise
```

Comment:
What does it raise?

Reply:
Oops, this would be much better off as an assert.